integration-tests/src/main/java/io/opentelemetry/SendTraceToJaeger.java

```diff
@@ -78,8 +83,8 @@ public static void main(String[] args) {
 
     // Start the example
     SendTraceToJaeger example = new SendTraceToJaeger(ip, port);
-    example.setupJaegerExporter();
-    example.myWonderfulUseCase();
+    OpenTelemetry openTelemetry = example.initOpenTelemetry();
+    SendTraceToJaeger.myWonderfulUseCase(openTelemetry);
```

Comment:
it's a little odd to use this class name to call a static method on itself, but it's not a deal breaker.

Reply:
you mean just write this as `myWonderfulUseCase(openTelemetry);` ?

Reply:
Error prone doesn't like

Reply:
Can you paste the error? I am curious

Reply:
Sorry my bad - error prone didn't like example.static(), but static() no problem. Error message earlier threw me off

```
/Users/aanuraag/git/opentelemetry-java/integration-tests/src/main/java/io/opentelemetry/SendTraceToJaeger.java:87: warning: [static] static method should be qualified by type name, SendTraceToJaeger, instead of by an expression
    example.myWonderfulUseCase(openTelemetry);
           ^
```